plugins/menufront.sma

```diff
@@ -142,15 +142,15 @@ stock bool:get_clcmd_flags(const search_command[], &flags)
 
 	return false;
 }
+
 public actionMenu(id, key)
 {
 	switch (key)
 	{
 		case 8: displayMenu(id, ++g_menuPosition[id])
 		case 9: displayMenu(id, --g_menuPosition[id])
-		default: client_cmd(id, "%s", g_menuCmd[g_menuPosition[id] * 8 + key])
+		default: client_cmd(id, "%s", g_menuCmd[g_menuPosition[id] * 8 + key]) // may need to replace : amxclient_cmd?
```

Comment:
If there are no plugins that rely on actually executing client commands, then `amxclient_cmd` should work just fine.